cdap-docs/admin-manual/source/system-requirements.rst

```diff
@@ -176,3 +179,27 @@ NTP (Network Time Protocol)
 - If you need to adjust the configuration (add or delete servers, use servers closer to you, etc.)::
 
     $ vi /etc/ntp.conf
+
+
+.. _admin-manual-cdap-and-firewalls:
+
+CDAP and Firewalls
+==================
+In your :ref:`cluster configuration <admin-manual-install-deployment-architectures>`, 
+there cannot be a firewall between the cluster and CDAP. Instead, if a
```

Comment:
Perhaps that diagram then needs editing.

One possibility is remove the firewall, from that diagram, and then add a discussion, with another diagram, showing the multiple locations for a firewall, and discussing the implications of the different choices and scenarios.

I can imagine some different configurations for that diagram that could include an "optional" firewall if we want to retain one in that illustration.